src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$hoverBlueColor: #00acdc;
+$hoverWhiteColor: #fff;
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  padding: 32px 0 16px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  width: 198px;
```

Comment:
You can use box-sizing: border-box; property and don't calculate manually width with borders